uefi-firmware.nix

```diff
@@ -48,11 +54,14 @@ let
     sha256 = "sha256-EPtI63jYhEIo4uVTH3lUt9NC/lK5vPVacUAc5qgmz9M=";
   };
 
-  _edk2-nvidia = fetchFromGitHub {
-    owner = "NVIDIA";
-    repo = "edk2-nvidia";
-    rev = "ad78b07af65d41bb96839f0bbe67bb445e04272f"; # Latest on r35.3.1-updates as of 2023-05-01
-    sha256 = "sha256-PdrisHYkmBXGvfkNboVvKJnBqORiM8sUsGySj7n5Y5c=";
+  _edk2-nvidia = applyPatches {
```

Comment:
Can you apply this in the `edk2-nvidia` applyPatches below instead of making a new one?

Reply:
Should be fixed now